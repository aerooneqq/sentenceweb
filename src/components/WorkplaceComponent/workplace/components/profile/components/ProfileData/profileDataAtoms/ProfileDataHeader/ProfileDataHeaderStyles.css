#profileDataHeader{ 
    width: 100%;
    height: 40px;
    background-color: #f0f0f0;

    border-style: solid;
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border-width: 0px;
    border-color: white;

    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: left;

    padding-left: 5%;

    font-family: Arial;
    color: #F97646;
    font-size: 20px;

    cursor: default;
}