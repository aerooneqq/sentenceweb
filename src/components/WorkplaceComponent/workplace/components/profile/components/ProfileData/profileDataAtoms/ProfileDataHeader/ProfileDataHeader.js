import React, {Component} from "react";

//Styles
import  "./ProfileDataHeaderStyles.css";

/**
 * PROPS LIST:
 * 1) dataName - the name of the currently opened data.
 */
export default class ProfileDataHeader extends Component { 
    render(){   
        return(
            <div id = "profileDataHeader">
                {this.props.dataName}
            </div>
        );
    }
}