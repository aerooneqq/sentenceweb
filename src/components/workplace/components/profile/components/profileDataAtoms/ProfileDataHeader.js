import React, {Component} from "react"

import  "./styles/ProfileDataHeaderStyles.css"

export default class ProfileDataHeader extends Component{ 
    constructor(props){ 
        super(props)
    }

    render(){ 
        return(
            <div id = "profileDataHeader">
                {this.props.dataName}
            </div>
        )
    }
}