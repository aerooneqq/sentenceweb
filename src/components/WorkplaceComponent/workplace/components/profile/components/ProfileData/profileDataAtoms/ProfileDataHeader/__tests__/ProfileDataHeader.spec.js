import React from "react";
import {shallow} from "enzyme"; 
import {shallowToJson} from "enzyme-to-json";
import ProfileDataHeader from "../ProfileDataHeader";

const props = { 
    dataName: "dataName"
}

describe("ProfileDataHeader", () => { 
    it("should render correctly", () => { 
        expect(shallowToJson(shallow(<ProfileDataHeader />))).toMatchSnapshot();
    });

    it("should display right header", () => { 
        expect(shallow(<ProfileDataHeader {...props}/>).find("div#profileDataHeader").text()).toBe(props.dataName);
    })
});