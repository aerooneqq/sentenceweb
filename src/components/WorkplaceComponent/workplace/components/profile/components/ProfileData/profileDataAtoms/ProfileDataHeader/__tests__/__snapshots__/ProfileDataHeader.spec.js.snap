// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileDataHeader should render correctly 1`] = `
<div
  id="profileDataHeader"
/>
`;
